src/store/reducers/hackathonReducer.js

```diff
@@ -0,0 +1,22 @@
+import * as ActionType from "../actions";
+
+const initialState = {
+  items: {},
```

Comment:
Is there only 1 hackathon?

Reply:
At the moment, there's only one created in Contentful, in the future I think admin can either unpublish or make the current hackathon content inactive in Contentful so the app will only fetch the active one.

Reply:
Thinking about the usability of that, I think it would be better if there could be multiple active hackathons, we just displayed the newest (latest `from` date) as the active one. it would open up the possibility of having a "previous hackathons" functionality. might be a bit big to change under this card, but might not be too bad.

Reply:
I just created another Hackathon content and found out that from the api call, the most recent one will always be at the first item in the array like items[0], would you still want us to create some kind of logic in the reducer to check the from date? or just loads all the hackathon events from the api call and only make the data available from the first item items[0]? thank you

Reply:
items[0] will be good enough for now, but just beware that admins will need to be aware that publishing a new hackathon will make it the active one, regardless of dates.